Mathlib/LinearAlgebra/FreeAlgebra.lean

```diff
@@ -8,6 +8,7 @@ import Mathlib.Algebra.FreeAlgebra
 import Mathlib.LinearAlgebra.FreeModule.Basic
 import Mathlib.LinearAlgebra.FinsuppVectorSpace
 import Mathlib.LinearAlgebra.FreeModule.StrongRankCondition
+import Mathlib.LinearAlgebra.Dimension.Constructions
 import Mathlib.LinearAlgebra.Dimension.StrongRankCondition
```

Comment:
This is tagged large-import but this file is almost a leaf file (only imported by `Mathlib.LinearAlgebra.TensorAlgebra.Basis`) and I think it's pretty reasonable to import `Algebra.adjoin` into it.

Reply:
Yes I agree. It's only two files that are seeing their imports increase by 60-70. I wouldn't worry much until they get used more